src/Orleans/Streams/PubSub/ImplicitStreamSubscriberTable.cs

```diff
@@ -9,10 +10,12 @@ namespace Orleans.Streams
     internal class ImplicitStreamSubscriberTable
     {
         private readonly Dictionary<string, HashSet<int>> table;
+        private readonly List<Tuple<IStreamNamespacePredicate, int>> predicates;
```

Comment:
This class needs be serializable.  I don't think this change takes that into account.